tests/test_bpo.py

```diff
@@ -267,3 +290,11 @@ def __init__(self, *, getitem=None):
     await bpo.router.dispatch(event, gh)
     assert gh.patch_data is None
     assert gh.patch_url is None
+
+
+def test_validate_issue_number():
```

Comment:
This is a "unit" test but it actually makes a call to bpo... maybe this should be mocked as well (or it may not be necessary all together).

Reply:
My preference is to mock the call to b.p.o.